### ISSUE TITLE
Documentation for OS Api GW as a cluster or multiple instances

### ISSUE THREAD

**Issue by brahama** [TykTechnologies/tyk#4026]
Hi everyone!

Basically i am doing a PoC to see if we can start using it in Production. BTW, i am loving all the power and things taht we can do! congrats :)

So a few questions that i could not find documentation for. I am running it in a K8s cluster, and everything runs correctly. I integrated it with the tyk operator and its marvelous.  BUT i did not find any information regarding using multiple tyk instances or cluster with just the GW. I saw there is a Sync CLI that could be used, but that removes the K8s operator of the picture.

Basically my question is where can we find documentation or example impementations where it is being isntalled as a clsuter or multiple instances and specially how to manage the API/Policies creation.

Is it the only way to use Sync? might be ppossible in the near future to use the K8s Operator ? Or the only way to have a cluster and synched is with the dashboard?

Thanks a lot for a great product!
Cheers

**Comment by caroltyk** [comment]
Please follow update via this issue: https://github.com/TykTechnologies/tyk-operator/issues/441
